src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+
+type Props = {
+  handleSubmit: (event: React.FormEvent) => void,
+  query: string,
+  setQuery: (title: string) => void,
+};
+
+export const Header:FC<Props> = ({
+  handleSubmit,
+  query,
+  setQuery,
+}) => (
+  <header className="todoapp__header">
+    {/* this buttons is active only if there are some active todos */}
+    {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Provide an `aria-label` attribute for the button to describe its purpose for better accessibility. 😇